app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -762,6 +756,20 @@ void OpenXRInputSource::Update(const XrFrameState& frameState, XrSpace localSpac
         return;
     }
 
+    auto isControllerUnavailable = [&poseLocation]() {
```

Comment:
I know that lambda closures  are elegant and the best approach in many cases, but I'm not sure this is one of those. Sorry if I'm missing something, but wouldn't this be equivalent to:

```
#if defined(PICOXR)
  bool isControllerUnavailable = true;
#else 
  bool isControllerUnavailable = (poseLocation.locationFlags & XR_SPACE_LOCATION_ORIENTATION_VALID_BIT) == 0
#endif
 if (isControllerUnavailable && GetHandTrackingInfo(frameState, localSpace, head)) {
        EmulateControllerFromHand(renderMode, head, delegate);
        return;
    }

```

Reply:
Yeah, I was thinking in using it in several places, but it ended up being just 1. I'll change it.